src/nft/components/sell/select/SelectPage.tsx

```diff
@@ -115,47 +150,169 @@ export const SelectPage = () => {
+      },
+      {
+        displayText: 'Date created',
+        onClick: () => {
+          setOrderBy(false)
+          setSortBy(SortBy.DateCreated)
+        },
+        icon: <PaintPaletteIconFilled width="28" height="28" color={vars.color.blue400} />,
+        reverseOnClick: () => setOrderBy(!orderByASC),
+      },
+      {
+        displayText: 'Date listed',
+        onClick: () => {
+          setOrderBy(false)
+          setSortBy(SortBy.DateListed)
+        },
+        icon: <TagFillIcon width="28" height="28" color={vars.color.blue400} />,
+        reverseOnClick: () => setOrderBy(!orderByASC),
+      },
+    ],
+    [orderByASC]
+  )
+
+  const SortWalletAssetsDropdown = () => <SortDropdown dropDownOptions={sortDropDownOptions} />
+
   return (
-    // Column style is temporary while we move over the filters bar that adjust width
-    <Column style={{ width: 'calc(100vw - 32px)' }}>
+    <Column width="full">
       <Row
         alignItems="flex-start"
         position="relative"
-        paddingLeft={{ sm: '0', md: '52' }}
+        paddingLeft={{ sm: '16', md: '52' }}
         paddingRight={{ sm: '0', md: '72' }}
         paddingTop={{ sm: '16', md: '40' }}
       >
-        <AnimatedBox paddingX="16" flexShrink="0" width="full">
-          <Row gap="8" flexWrap="nowrap">
-            <CollectionSearch searchText={searchText} setSearchText={setSearchText} />
-            <SelectAllButton />
-          </Row>
-          <InfiniteScroll
-            next={fetchNextPage}
-            hasMore={hasNextPage ?? false}
-            loader={
-              hasNextPage ? (
-                <Center>
-                  <LoadingSparkle />
-                </Center>
-              ) : null
-            }
-            dataLength={displayAssets.length}
-            style={{ overflow: 'unset' }}
+        <FilterSidebar SortDropdown={SortWalletAssetsDropdown} />
+
+        {(!isMobile || !isFiltersExpanded) && (
+          // @ts-ignore
+          <AnimatedBox
+            paddingLeft={isFiltersExpanded ? '24' : '16'}
+            flexShrink="0"
+            style={{
+              transform: gridX.interpolate(
+                (x) => `translate(${parseInt(`${x}`) - (!isMobile && isFiltersExpanded ? 300 : 0)}px)`
+              ),
+              width: gridWidthOffset.interpolate((x) => `calc(100% - ${x}px)`),
+            }}
           >
-            <div className={assetList}>
-              {displayAssets && displayAssets.length
-                ? displayAssets.map((asset, index) => <WalletAssetDisplay asset={asset} key={index} />)
-                : null}
-            </div>
-          </InfiniteScroll>
-        </AnimatedBox>
+            <Row gap="8" flexWrap="nowrap">
+              <FilterButton
+                isMobile={isMobile}
+                isFiltersExpanded={isFiltersExpanded}
+                results={displayAssets.length}
+                showFilterBadge={collectionFilters.length > 0}
```

Comment:
yeah I removed this as a prop it checks it itself now

Reply:
I added the case for showing badge to the Filter button. How much did removing the prop save complexity because it seems like we're over-fitting the filter button by adding these multiple checks to the component.